xmtp/README.md

```diff
@@ -79,17 +79,17 @@ processMessages():
             If user.last_refreshed is uninitialized or more than THRESHOLD ago:
                 refreshUserInstallations()    // Could be kicked off asynchronously or synchronously
                 return  // refreshUserInstallations() will call back into processMessages() when ready
-        Fetch the installations of all users from the DB
+        Fetch the installations and sessions of all users from the DB
         For each installation:
+            If installation.state == UNINITIALIZED:
+                Create an outbound session (hold it in memory)
```

Comment:
Verifying that you want the Created Session to no be persisted. I had assumed that New sessions would be persisted, however they would not be ratcheted until line 93. Happy either way